downloader/src/main/kotlin/vcs/Subversion.kt

```diff
@@ -183,23 +183,36 @@ object Subversion : VersionControlSystem() {
                     getWorkingTree(File(targetDir, pkg.vcsProcessed.path))
                 }
             } else {
-                log.info { "Trying to guess a $this revision for version '${pkg.id.version}'." }
+                val tagPath: String
+                val path: String
 
-                revision = getWorkingTree(targetDir).guessRevisionName(pkg.id.name, pkg.id.version)
+                if (pkg.vcsProcessed.path.startsWith("tags/")) {
+                    log.info { "Ignoring the $this revision as the path points to a tag." }
```

Comment:
Consider `...revision '${pkg.vcsProcessed.revision}' as the...`.

Reply:
Actually, in this case the revision is always empty, as otherwise the above

    if (allowMovingRevisions || isFixedRevision(revision)) {

branch is taken.

Reply:
Ah, not true, it could be `HEAD`. Will make the change.